host/src/global.json

```diff
@@ -0,0 +1,7 @@
+{
+  "sdk": {
+    "version": "9.0.100-preview.6.24328.19",
```

Comment:
There is an rc version now, do we want to move off preview? `9.0.100-rc.1`

Reply:
I still do not see rc1 in the ADO task. The latest it is still bringing is preview.6.

Reply:
I found 9.0.100-rc.1.24452.12 available now. Will switch to that.

Reply:
The platform may not have the latest bits when this version gets deployed (platform always lags behind). So, it's safe to stick with preview6 for now. Can probably update to rc1/2 in the next version of host.